openapi/data/fields.py

```diff
@@ -222,9 +222,21 @@ def dump(self, value):
 
 
 class DecimalValidator(NumberValidator):
-    def __call__(self, field, value, precision):
+    def __call__(self, field, value):
         try:
             value = Decimal(value)
         except TypeError:
             raise ValidationError(field.name, '%s not valid Decimal' % value)
-        return super().__call__(field, value, precision)
+        return super().__call__(field, value)
+
+
+class BoolValidator(Validator):
+
+    def __call__(self, field, value):
+        value = str(value).lower()
+        if value not in ('true', 'false'):
+            raise ValidationError(field.name, '%s not valid' % value)
+        return value == 'true'
```

Comment:
For other validators we return the value, but for this one we do a comparison. Is this what we want?

Reply:
The comparison will return `True` or `False`